src/pkg/control/repository/repo.go

```diff
@@ -51,3 +51,25 @@ const (
 	// https://github.com/kopia/kopia/blob/f9de453efc198b6e993af8922f953a7e5322dc5f/repo/maintenance/maintenance_safety.go#L42
 	NoMaintenanceSafety
 )
+
+type RetentionMode int
+
+// Can't be reordered as we rely on iota for numbering.
```

Comment:
I'm (still) increasingly wary of iota, for exactly this reason.  I just wrapped another PR where the author tried to slip a value into the middle of an iota enumeration.  Purely an accident, but the sort of accident that's easy to make and difficult to catch if it goes through (short of writing equivalence unit tests).

At minimum, I think an equivalence unit test is warranted for any iota enumeration, for protection.  Or, alternatively, we could use hardcoded ints instead of the iota.